.github/workflows/CI.yml

```
@@ -56,6 +56,10 @@ jobs:
           issue_labels: vulnerability
           dedupe_issues: true
 
+      - name: npm run check
```

A bit confusing to have the job's name same as the command it's running IMO. 
```suggestion
      - name: Run linters
```